### ISSUE TITLE
A friendly comment

### ISSUE THREAD

**Issue by wangheer2010** [evidentlyai/evidently#13]
Hi developers👋 , thank you so much for developing the amazing package :) It is easy to use and really useful!
I have an idea of making a small change. In real business, we usually use historical data as the "reference" and the current period data as "current". That is to say, we are likely to have a much larger amount of data for "reference" comparing to "current".
For the CatTargetDrift, I think it would be better if you could add histnorm='probability density' to the Target (Predict) By Feature plots.
Thank you for your time and consideration.

**Comment by emeli-dral** [comment]
Hi! Thank you so much for your feedback, we appreciate it a lot☺️

This is very nice suggestion, to be honest we also considered ```percent``` and ```probability density``` options to have percentage or fraction of samples instead of the count of samples. 

Personally, I prefer to use the ```probability density``` normalisation for my projects. The reason to choose ```count``` was that it is easier to translate the count of samples into percent or fraction than vice versa🤷‍♀️  Do you think there are additional arguments in favor of  ```probability density``` I missed? We are considering changing it, but look to collect more feedback from the users on what is the most useful view. 

We also think that it will be good to have this as an option. We hope to implement the service version for our tool and add a lot of parameters to customise the reports.

**Comment by gakuba** [comment]
Hi @emeli-dral, I tested the package on one of the datasets that I am using for a tutorial that I am preparing for my blog. the training set has a shape of (243728, 20) and the validation set has (91493, 20). I found that the generated HTML file could take up to 4 min to open in the browser when the report contains DriftTab only and it crashed when I added another tab on the report. I am using a MacBook Pro with 16GB of RAM. the browser is chrome.
